src/app/config/projectProperties.json

```diff
@@ -0,0 +1,68 @@
+{
+    "projectTypes": {
+        "angular": {
+            "name": "angular",
```

Comment:
Each property within "projectTypes" is the name of the project type, so it is redundant to have it also have the same thing as the "name" property within the object.

Reply:
Fixed locally